Library/Homebrew/cmd/info.rb

```diff
@@ -165,9 +165,9 @@ def info_formula(f)
 
   def decorate_dependencies(dependencies)
     deps_status = dependencies.collect do |dep|
-      dep.installed? ? pretty_installed(dep) : pretty_uninstalled(dep)
+      dep.satisfied?([]) ? pretty_installed(dep_display_s(dep)) : pretty_uninstalled(dep_display_s(dep))
```

Comment:
What's the right style for a muti-line trinary expression? Or should it be converted to an `if ... else ... end`?